heroku_connect/checks.py

```diff
@@ -33,18 +35,19 @@ def _check_foreign_key_target(app_configs, **kwargs):
 
 def _check_unique_sf_object_name(app_configs, **kwargs):
     errors = []
-    models = set(get_heroku_connect_models())
-    unique_models = set({
-        model.sf_object_name: model
-        for model in models
-    }.values())
-
-    for model in models - unique_models:
-        errors.append(Error(
-            "%s.%s.sf_object_name clashes with another model." % (
-                model._meta.app_label, model.__name__),
-            hint="Specify a unique 'sf_object_name' argument.",
-            id='heroku_connect.E006',
-        ))
+    model_map = defaultdict(list)
+    for model in get_heroku_connect_models():
+        model_map[model.sf_object_name].append(model)
+
+    for sf_object_name, models in model_map.items():
+        if len(models) > 1:
+            for model in models:
+                errors.append(Error(
+                    "%s.%s.sf_object_name clashes with another model." % (
```

Comment:
I honestly thing the error can be clearer if we say which model it clashes with, not just `another model` and since you have it in the list, I think it's just better to list all the models that are clashing together...

Reply:
I agree... but it could clash with N other models. That being said, I raise an error for each of them, therefore you will have a list of all models that clash to improve debugging.

Reply:
Then I would say at least add the `sf_object_name` to the error...so that we can decouple the clashing ones, which makes it even better for debugging...

Reply:
done 👍